src/lints/exported_function_changed_abi.ron

```diff
@@ -0,0 +1,42 @@
+SemverQuery(
+    id: "function_changed_export_name",
+    human_readable_name: "TODO",
+    description: "TODO",
+    required_update: Major,  // TODO
+    reference_link: None,  // TODO
```

Comment:
Suggestion to fill in these TODOs, based on how the other fn ABI lint had them: https://github.com/obi1kenobi/cargo-semver-checks/blob/main/src/lints/function_changed_abi.ron

```suggestion
    human_readable_name: "exported function changed ABI",
    description: "A function marked `#[no_mangle]` or assigned an explicit `#[export_name]` changed its external ABI.",
    required_update: Major,
    reference_link: Some("https://doc.rust-lang.org/reference/items/external-blocks.html#abi"),
```